files/pages/index.js

```diff
@@ -6,7 +6,7 @@ import Banner from "../components/Banner";
 import Footer from "../components/Footer";
 import BlogHome from "../components/BlogHome";
 import CTAThree from "../components/CTAThree";
-import Contact from "../components/Contact";
```

Comment:
Changes aren't expected in this file
only in invitation related files must be there